src/app/upgrade/upgrade.component.html

```diff
@@ -22,9 +22,9 @@
       </mat-card-header>
       <mat-card-content>
         <p i18n>
-            The upgrade of the system has been done successfully.<br>
-            Please restart your system to apply changes
+          The upgrade of the system has been done successfully.<br>
```

Comment:
Could you remove the `<br>` tag?  Using `<p>` tags is preferred, anyway, so this is a redundant extra space.